pkg/controller/customdomain/customdomain_controller.go

```diff
@@ -142,9 +142,9 @@ func (r *ReconcileCustomDomain) Reconcile(request reconcile.Request) (reconcile.
 // modifyClusterDomain modifies the cluster domain
 func modifyClusterDomain(r *ReconcileCustomDomain, reqLogger logr.Logger, instance *customdomainv1alpha1.CustomDomain, finalize bool) error {
 	var (
-		domain        = ""
-		cert          = ""
-		crNeedsUpdate = false
+		cert          string
```

Comment:
The assignments with empty string are redundant because when declaring a variable in go those are the default values.